### PR TITLE
🌱 initial implementation of base application (iss. #1)

### LINKED ISSUE #1: base application with peer review comment query
Help M-Write researchers transition from using the [M-Write Peer Review](/M-Write/mwrite-peer-review) (MPR) LTI application to Canvas' built-in peer review features.  Develop an ETL application to extract Canvas peer review data and store it in a database.  The database may use a schema similar to that of MPR in order to make the transition easier.  This application is needed because Unizin's UDP and UDW (deprecated) do not already have this data.

### PR DESCRIPTION
**Pull request by lsloan**
Resolves #1.

Base application structure and execution in place.  Uses parameters from environment variables to query an assignment for peer review comments and save the raw data to a JSON file.

## Test plan

1. Copy `config/.env.sample` to `.env` in the root of the project.  Edit the file to replace the values contained therein with valid ones for the test environment.  

    ```sh
    cp config/.env.sample .env
    vim .env
    ```

1. Use Docker's `compose` tool to build the application, run it, and produce an output file.

    ```sh
    docker compose up --build
    ```

1. Examine the data in the output file, `assessments.json`.

### CODE REVIEW

**Comment by lsloan** [comment]
@zqian: This PR isn't ready for review, but I know you like to see how the project is progressing.  You can give suggestions if you like, but keep in mind this is a work in progress.

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      - ./mysql:/docker-entrypoint-initdb.d:ro
+    container_name: peer-review-data_db
+  main:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    depends_on:
+      - db
+    env_file:
+      - .env
```
Comment (zqian) [inline docker-compose.yaml:29]: Need to provide a sample .env file. I cannot get docker-compose started for testing.

**Review by zqian — CHANGES_REQUESTED** [review]

**Comment by zqian** [comment]
I can test the code now with params for Canvas test server. I think the test plan instructions in this PR can be added into the README file.

**Comment by lsloan** [comment]
@zqian, would updates to the README be the key for you to give this PR an approving review?

**Comment by lsloan** [comment]
@zqian, please review again. Last time you mentioned it'd be good to have more in the `README`, but I wasn't sure whether that was a comment or a requirement for PR approval.  I've not gotten any messages about it since then, so I decided to treat it as a requirement.

Please review again and give an approval so I can merge the PR or give specific feedback on what's required to make this PR mergeable.

Thanks!

**Comment by zqian** [comment]
Thanks for updating the README file. I will approve the PR.

**Review by zqian — APPROVED** [review]

**Pull request merged**